disnake/abc.py

```diff
@@ -959,10 +959,10 @@ async def move(
         self,
         *,
         after: Snowflake,
-        offset: int = MISSING,
-        category: Optional[Snowflake] = MISSING,
-        sync_permissions: bool = MISSING,
-        reason: str = MISSING,
+        offset: int = ...,
+        category: Optional[Snowflake] = ...,
+        sync_permissions: bool = ...,
+        reason: Optional[str] = ...,
```

Comment:
reason type in docstring should be optional as well

Reply:
https://github.com/DisnakeDev/disnake/pull/482/commits/51c5f84eaceae0187de2843967e4eb74da14b201